eventhub/Module.md

```diff
@@ -1,55 +1,22 @@
-# Ballerina Azure Event Hubs Module
+## Overview
 
-Connects to Microsoft Azure Event Hubs using Ballerina.
+Azure Event Hub Ballerina Connector is used to connect with the Azure Event Hubs via Ballerina language easily 
```

Comment:
Suggestion: "The Azure Event Hub Ballerina Connector is used to connect Ballerina to the Azure Event Hub to ingest, buffer, store, and process high-frequency data from any source in real-time to derive important business insights."

Reply:
Fixed in df7c75b . Thank you.